README.md

```diff
@@ -1,3 +1,46 @@
+# Music Library
```

Comment:
There are some important bits of info missing from this. Some things that would make it much better:
- some instructions regarding the database and the need to create environment variables,
- what technologies are required to use it, what the application dependencies are (`Express` and `MySQL2`), and what the development dependencies are (`nodemon`, `dotenv`, `mocha`, `chai`, `superTest`)
- more detailed information about the routes provided by the app, what type of HTTP verb they all require and an example request body would be a great addition,
- how to test the app,
- what you would do if you had more time (if applicable)